pkg/controller/servicediscovery/servicediscovery_controller.go

```diff
@@ -2,8 +2,10 @@ package servicediscovery
 
 import (
 	"context"
+	stderr "errors"
```

Comment:
nit:

  can we rename this to "goerrors" or something similar?

stderr is an specific name for the stderr output, I thought the code was wrong when I saw it :)

Reply:
Done. :)